collectors/persistentvolumeclaim.go

```diff
@@ -87,9 +98,26 @@ func (collector *persistentVolumeClaimCollector) Collect(ch chan<- prometheus.Me
 	}
 }
 
+// getPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
+func getPersistentVolumeClaimClass(claim *v1.PersistentVolumeClaim) string {
+	// Use beta annotation first
+	if class, found := claim.Annotations[v1.BetaStorageClassAnnotation]; found {
```

Comment:
hi, @andyxning
[GetPersistentVolumeClaimClass](https://github.com/kubernetes/client-go/blob/v4.0.0-beta.0/pkg/api/helper/helpers.go#L549) in client-go use `k8s.io/client-go/pkg/api.PersistentVolumeClaim`, but we all use types under `k8s.io/client-go/pkg/api/v1` package. Types mismatch.